Jenkinsfile

```diff
@@ -19,15 +20,17 @@ pipeline {
             agent { label 'dockerhost' }
             steps {
                 echo 'Building...'
-                sh "docker build -t kiva/kiosa-dev -f Dockerfile.prod ."
+                sh "docker build -t kiva/kiosa-dev:${TAG_NAME} -f Dockerfile.prod ."
             }
         }
         stage('Publish') {
             agent { label 'dockerhost' }
             steps {
                 echo 'Publishing...'
+                // are we assuming that the publish steps happens on the same node as the build step?
```

Comment:
Yes, I agree. This works right now because we have one dockerhost agent available. I am indifferent right now to whether we collapse the stages or require the same node across multiple stages, because, to me, this is starting to push against larger issues with our current Jenkins + Docker strategy. Making some strategic changes would end up implying some changes to our pipeline stages.

So I'm OK with either improvement (collapse stages or require same node), and I'll write up a ticket for the Docker + Jenkins generally piece.